lib/pbench/server/api/resources/users_api.py

```diff
@@ -470,24 +470,23 @@ def delete(self, username):
             abort(403, message="Not authorized to delete user")
 
         try:
-            user = User.query(username=username)
-            # Do not delete if the user is admin
-            if not user.is_admin():
-                User.delete(username)
+            User.delete(username)
+            self.logger.info("User entry deleted for user with username {}", username)
```

Comment:
When we look in the log, will we have enough context to determine the user who made the request?  (Or, should we attach that to this message?)

Reply:
yeah I also added by which user this action was performed (meaning whose auth id it was), do you think its enough information which user got deleted and by whom?